strings/valentines.json

```diff
@@ -73,11 +75,26 @@
 		"Are you wearing space pants? I wish you weren't.",
 		"You give me a positive moodlet.",
 		"Let's make like swarmers and reproduce.",
-		"If you were a drone, would you be free tonight?"
+		"If you were a drone, would you be free tonight?",
+		"Get the paramedics -- your body put me into crit!",
+		"Come to the incinerator and be my maint squeeze.",
+		"Come to the courtroom so we can spend some time alone.",
+		"I'm the gray tide -- may I take your hand tele?",
+		"If I were a clown car, would you come inside me?",
+		"If you were telecoms, would you go down on me?",
+		"Your face gets me higher than omega weed.",
+		"Someone check the tachyon doppler array, because you just rocked my world.",
+		"I wish I were your family heirloom so I could be with you all the time.",
+		"How much damage does a toxins bomb do? Enough to break the ice.",
+		"Call me a gibtonite deposit, because I've got a lot of love to gib tonite.",
+		"I don't need an ion cannon, I've got my ion you.",
+		"I don't need the chef's cooking when you're such a snack!",
```

Comment:
This might read better as "I don't need to be a cook to know you're a snack!"

Reply:
I mean everyone can tell what's a snack, it has different in-game effects than chef food /overthinking

Reply:
OH! I misinterpreted the line. This is great. Nevermind.